xc/common/utils/prjuray_timing.py

```diff
@@ -0,0 +1,530 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Why does this need a full copy/paste?

Reply:
I'm not entirely happy with that but I didn't wanted any U-Ray scripts depend on X-Ray stuff.

The content of this file is copied from `third_party/prjxray/prjxray/timing.py`. This code is not present in `prjuray-tools` so I couldn't import it from there. Since it is not directly related to X-Ray/U-Ray database but rather to how it is handled in arch-defs I decided to put it in arch-defs, at least for U-Ray import for now.

It may be worth to rename this to eg. `timing.py` and make X-Ray import use it as well since it is common to both archs. Any thoughts on that?

Reply:
We don't have a uray routing timing model just yet, so can we stub this out, rather than assuming the 7-series and US+ route timing model will be the same?